src/vm/bytecode/isa.lisp

```diff
@@ -199,3 +199,12 @@
 (defmethod print-isa ((operand isa-operand) &optional (stream t))
   (with-slots (name) operand
     (format stream "~A" name)))
+
+
+;; executes body in a context where the opcode is bound to a symbolic name matching the mnemonic in the isa
+;; we use symbol macros to make this work
+(defmacro with-opcodes-from-current-isa (&body body)
```

Comment:
Currently the Isa definition isn’t using eval-when but this macro needs access to the current-isa. I think we need to change that to make sure this works always